src/content/tools/pub/automated-publishing.md

```diff
@@ -68,7 +68,7 @@ If you have sufficient permission, you can enable automated publishing by:
 
 The _repository_ is the `<organization>/<repository>` on GitHub.
 For example, if your repository is
-`https://github.com/dart-lang/pana` you must specify `dart-lang/pana` in the
+`{{site.repo.dart.org}}/pana` you must specify `dart-lang/pana` in the
```

Comment:
Hmm this change and the other in the file aren't correct as they are meant to be the literal URL. Could the tool or the config ignore values within inline code and code blocks?

Reply:
Ignored in the configuration ([/tool/config/site_variable_scanner.yaml](https://github.com/dart-lang/site-www/blob/aa74188529d8af652914929e27243a2ef00cdf94/tool/config/site_variable_scanner.yaml#L67))